single_subject/batch_single_subject.sh

```diff
@@ -402,5 +405,49 @@ sct_smooth_spinalcord -i t1.nii.gz -s t1_seg.nii.gz
 sct_flatten_sagittal -i t1.nii.gz -s t1_seg.nii.gz
 # Note: Use for visualization purposes only
 
+
+
+# New features (SCT v6.5, December 2024)
+# ======================================================================================================================
+
+# Lesion analysis for SCI and MS lesions
+cd ../t2_lesion
+# Segment the spinal cord and intramedullary lesion using the SCIsegV2 model
+# Note: t2.nii.gz contains a fake lesion for the purpose of this tutorial
+sct_deepseg -i t2.nii.gz -task seg_sc_lesion_t2w_sci -qc ~/qc_singleSubj
+# Note: Two files are output:
+# - t2_sc_seg.nii.gz: the spinal cord segmentation
+# - t2_lesion_seg.nii.gz: the lesion segmentation
+# Check results using FSLeyes
+fsleyes t2.nii.gz -cm greyscale t2_sc_seg.nii.gz -cm red -a 70.0 t2_lesion_seg.nii.gz -cm blue-lightblue -a 70.0 &
+# Compute various morphometric measures, such as number of lesions, lesion length, lesion volume, etc.
+sct_analyze_lesion -m t2_lesion_seg.nii.gz -s t2_sc_seg.nii.gz -qc ~/qc_singleSubj
+
+# Lesion analysis using PAM50 (the -f flag is used to specify the folder containing the atlas/template)
+# Note: You must go through the "Register to Template" steps (labeling, registration) first
+# sct_analyze_lesion -m t2_lesion_seg.nii.gz -s t2_sc_seg.nii.gz -f label -qc ~/qc_singleSubj
+
+# Note: We also have a contrast-agnostic segmentation command for MS lesions, too:
+sct_deepseg -i t2.nii.gz -task seg_ms_lesion -qc ~/qc_singleSubj
+# As well as a segmentation command tailored to MP2RAGE MS lesions
+sct_deepseg -i t2.nii.gz -task seg_ms_lesion_mp2rage -qc ~/qc_singleSubj
+
+# Rootlets segmentation
+cd ../t2
+# Segment the spinal nerve rootlets
+sct_deepseg -i t2.nii.gz -task seg_spinal_rootlets_t2w -qc ~/qc_singleSubj
+# Check results using FSLeyes
+fsleyes t2.nii.gz -cm greyscale t2_seg.nii.gz -cm subcortical -a 70.0 &
```

Comment:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/f226d04768b64d5c68e00a442f209de1eea153cd/CHANGES.md?plain=1#L31

```suggestion
fsleyes t2.nii.gz -cm greyscale t2_rootlets.nii.gz -cm subcortical -a 70.0 &
```